t/02-integration/vary.t

```diff
@@ -445,7 +450,11 @@ location /vary8_prx {
 
 location /vary {
     content_by_lua_block {
-        local incr = ngx.shared.ledge_test:incr("test8", 1, 0)
+        local incr = ngx.shared.ledge_test:incr("test8", 1)
+        if not incr then
+            incr = ngx.shared.ledge_test:incr("test8", 1, 0)
```

Comment:
What's this change for? Why manually check for a failing `incr` without an init argument and then call it again with an init argument? Seems like the behaviour should be identical but this has 3 extra lines?

Reply:
Ah yes, I'll revert that. I was getting odd results with the counters and glancing at the docs I (incorrectly) thought it was always re-initialising. I meant to take it out again...